app/src/androidTest/java/com/epfl/drawyourpath/database/MockDatabaseTestPhoto.kt

```diff
@@ -16,8 +15,8 @@ class MockDataBaseTestPhoto {
      */
     @Test
     fun setProfilePhotoCorrectly() {
-        val database = MockDataBase()
-        val isSet = database.setProfilePhoto(photoProfile).get()
-        Assert.assertEquals(database.userIdToUserAccount.get(userIdTest)?.getProfilePhoto(), photoProfile)
+        val database = MockDatabase()
+        database.setProfilePhoto(userIdTest, photoProfile).get()
+        // Assert.assertEquals(database.users[userIdTest].picture, photoProfile)
     }
```

Comment:
While refactoring I disabled photos because we are currently storing them in database and they shouldn't.

Reply:
I don't think that we will have the time to implement them on the Firebase. Since the test are already implemented, I think it's better to keep this test. It will improve the code coverage and check that this current implementation is working as expected.